src/components/components/Component.js

```diff
@@ -89,24 +126,29 @@ export default class Component extends React.Component {
     }
   };
 
+  selectOption = objName => {
+    const value = this.state.nameToLinkMap[objName];
+    updateEntity.apply(this, [this.props.entity, this.props.name, value]);
+  }
+
   /**
    * Render propert(ies) of the component.
    */
   renderPropertyRows = () => {
     const componentData = this.props.component;
 
     if (isSingleProperty(componentData.schema)) {
-      const componentName = this.props.name;
-      const schema = AFRAME.components[componentName.split('__')[0]].schema;
       return (
-        <PropertyRow
```

Comment:
We shouldn't modify all components right? Shouldn't we leave the original stuff that is present in the original aframe-inspector and only modify if the component that is being added is `gltf-model`?
For example, if you try to select `environment` in regular student scene it works. But doing this in our version of aframe-inspector, it breaks. I think this is because of this change.